contrib/mixin/mixin.libsonnet

```diff
@@ -360,7 +362,7 @@
               steppedLine: false,
               targets: [
                 {
-                  expr: 'sum(rate(grpc_server_started_total{job="$cluster",grpc_type="unary"}[5m]))',
+                  expr: 'sum(rate(grpc_server_started_total{%s="$cluster",grpc_type="unary"}[5m]))' % $._config.clusterLabel,
```

Comment:
Grafana dashboards are not my thing so no clue about this change, reads a bit odd to me @brancz can you have a look as well, thanks!

Reply:
Anything I can do to clarify this? It's a format string setting the label key to $._config.clusterLabel

Reply:
All good, just wanted second eyes on this. Thanks!